src/classes/UTIL_SortContacts_TEST.cls

```diff
@@ -0,0 +1,193 @@
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2019
+* @group Utilities
+* @description Test for UTIL_SortContacts
+*/
+@isTest
+private class UTIL_SortContacts_TEST {
+	/*********************************************************************************************************
+    * @description Test Method to test sorting Contacts alphabetically by LastName FirstName
+    */
+    @isTest()
+    private static void sortContactsLastFirstName() {
+        List<UTIL_SortContacts> utilSortCons = new List<UTIL_SortContacts>();
+        List<Contact> contacts = UTIL_UnitTestData_TEST.getMultipleTestContacts(9);
+
+        contacts[0].FirstName = 'Will';
+        contacts[0].LastName = 'Zap';
+        contacts[1].FirstName = 'William';
+        contacts[1].LastName = 'Zap';
+        contacts[2].FirstName = 'Abigail';
+        contacts[2].LastName = 'Zap';
+        contacts[3].FirstName = 'Tammy';
+        contacts[3].LastName = 'Brett';
+        contacts[4].FirstName = 'Betty';
+        contacts[4].LastName = 'Brett';
+        contacts[5].FirstName = 'Charles';
+        contacts[5].LastName = 'Brett';
+        contacts[6].FirstName = 'Cat';
+        contacts[6].LastName = 'Pierre';
+        contacts[7].FirstName = 'Lyla';
+        contacts[7].LastName = 'Pierre';
+        contacts[8].FirstName = 'Annie';
+        contacts[8].LastName = 'Pierre';
+
+        Test.startTest();
+        utilSortCons.add(new UTIL_SortContacts(contacts[0]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[1]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[2]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[3]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[4]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[5]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[6]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[7]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[8]));
+        Test.stopTest();
+        utilSortCons.sort();
```

Comment:
I always mess this up so I may be wrong here, but do we want the sort() call before stop test? Probably not a big deal either way and like I said, I can never remember anyway so I could be wrong.

Reply:
I've moved the sort() inside the test.startTest()/test.stopTest().